packages/eslint-plugin/src/configs/recommended.json

```diff
@@ -5,6 +5,7 @@
     "@angular-eslint/component-class-suffix": "error",
     "@angular-eslint/contextual-lifecycle": "error",
     "@angular-eslint/directive-class-suffix": "error",
+    "@angular-eslint/no-duplicates-in-metadata-arrays": "error",
```

Comment:
Please revert, we don't make changes to recommended configs outside of major versions.

Related: did you manually add this? Because I don't see any recommended metadata on the rule? If we are generating here by default then that's a bug in the generation script IMO

Reply:
Reverted. I must've done this manually forgetting that we don't do this outside of major versions. I ran `update-rule-lists` and it did not update this file.